charts/portal/README.md

```diff
@@ -210,6 +210,7 @@ This section describes configurable parameters in **values.yaml**, there is also
 | `portal.defaultTenantId` | **Important!** Do not change the default tenant ID unless you have been using a different tenant ID in your previous install/deployment. There is a 15 character limit. See [DNS Configuration](#dns-configuration) for tenant ID character limitations.  | `apim` |
 | `portal.registryCredentials` | Used to create image pull secret, see prerequisites | `` |
 | `portal.useExistingPullSecret` | Configures Portal Deployment to use **global.pullSecret** as imagePullSecret | `false` |
+| `portal.useExistingDBSecret` | Configures Portal Deployment to use **global.databaseSecret** for fetching the DB password | `false` |
```

Comment:
Is this applicable for the demo DB? Also, its better to have this under global along with other db related configuration.

Reply:
@APIM-KiranVaddadi I had thought about it. Image pull secret we handled in 'portal' section, so followed the same.